### ISSUE TITLE
Review documentation process and items from #55

### ISSUE THREAD

**Issue by douglas-reid** [istio/api#56]
There were a few open issues that we deferred from PR #55 to unblock progress (for pending demos) and the like. We should revisit these issues (and document generation) at a later date.

**Comment by geeknoid** [comment]
Doug, could you decide what to do with this?

**Comment by sakshigoel12** [comment]
@douglas-reid
